presentation/src/main/java/com/mashup/alcoholfree/presentation/ui/splash/SplashActivity.kt

```diff
@@ -0,0 +1,53 @@
+package com.mashup.alcoholfree.presentation.ui.splash
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.core.splashscreen.SplashScreen.Companion.installSplashScreen
+import androidx.core.view.WindowCompat
+import com.kakao.sdk.auth.AuthApiClient
+import com.kakao.sdk.user.UserApiClient
+import com.mashup.alcoholfree.presentation.ui.home.HomeActivity
+import com.mashup.alcoholfree.presentation.ui.login.LoginActivity
+import com.mashup.alcoholfree.presentation.ui.theme.AlcoholFreeAndroidTheme
+
+class SplashActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        installSplashScreen()
+
+        super.onCreate(savedInstanceState)
+        WindowCompat.setDecorFitsSystemWindows(window, false)
```

Comment:
```suggestion
        installSplashScreen()
        super.onCreate(savedInstanceState)
        
        WindowCompat.setDecorFitsSystemWindows(window, false)
```

Nit: 엔터 불편하네요

그리고 installSplashScreen은 super.onCreate()보다 먼저 호출되어야하나보네..!

Reply:
마자용 먼저 호출되어야 한대용 엔터 고칠게욬ㅋㅋㅋㅋㅋㅋㅋㅋ

Reply:
암이 낳앗습미다